mne/io/eeglab/eeglab.py

```diff
@@ -6,11 +6,14 @@
 import os.path as op
 
 import numpy as np
+import h5py  # Added to read newer Matlab files (7.3 and later)
```

Comment:
You don't need to update the CIs, we already use `h5py` elsewhere. Just nest as suggested and add `requires_h5py` decorator to applicable tests that need `h5py`